go.mod

```diff
@@ -277,5 +277,6 @@ replace (
 	// fix potential security issue(CVE-2020-26160) introduced by indirect dependency.
 	github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.6-0.20210809144907-32ab6a8243d7+incompatible
 	github.com/pingcap/tidb/parser => ./parser
+	github.com/pingcap/tipb => github.com/L-maple/tipb v0.0.0-20230313092931-abd3eac30f1a
```

Comment:
tipb: https://github.com/pingcap/tipb/pull/297?w=1 have to be merged first.

Reply:
@Defined2014 @mjonss  Please help me review the tipb PR. If you have any questions, please comment and I would reply as quickly as possible, thanks~

Reply:
I think we split the optimization of pushing down the IndexFilter / Selection into a future PR.

Reply:
@mjonss For global index, pushing down `partitionIds` is not just an optimization. If `partitionIds` is not pushed down, there are wrong query cases.  I mention my questions(or say, awkward situation) in tipb PR.